smartseq2_single_sample/check_reference/buildRef.wdl

```diff
@@ -0,0 +1,20 @@
+import "BuildStarReferenceBundle.wdl" as buildStarRef
+
+workflow buildStarRefs {
+  File ref_metadata_file
+  Array[Object] metadata = read_objects(ref_metadata_file)
+  
+  scatter (record in metadata){
+    String name_ref = record.name1
+    String name2 = record.name2
+    File fasta = record.fasta
+    File gtf = record.gtf
+    
+    call buildStarRef.BuildStarReference as StarRef {
+      input:
+        ref_fasta = fasta,
+        gtf_file = gtf,
+        ref_name = name_ref
```

Comment:
I feel like inverting the name in this way (`ref_name <-> name_ref`) makes downstream maintenance a little trickier. Maybe the SWE who reviews this can comment.